src/man/include/ad_modified_defaults.xml

```diff
@@ -92,6 +92,18 @@
                     this fallback behavior, you can explicitly
                     set "fallback_homedir = %o".
                 </para>
+                <para>
+                    Note that the system typically expects home directory
```

Comment:
I would suggest adding "a" before home directory here.  like this:

"expects a home directory"